scss/jquery/overrides/_ui-dialog.scss

```diff
@@ -3,3 +3,23 @@
 .ui-dialog {
   @include ui-dialog;
 }
+
+#{$outside-namespace-marker} {
+  .crm-container.ui-dialog {
+    &.fullscreen {
+      bottom: 0;
+      max-height: none !important;
```

Comment:
Same here.

Reply:
This is actually not required, thanks for flagging. Removed.